### DISCUSSION TITLE
Please provide a method to get the build URL

### DISCUSSION THREAD

**Discussion by MartinP-Autopilot** [percy/percy-playwright#51] [Ideas]
As we are running these in CICD, it would be good to be able to print the URL to the console so that we can easily navigate from the CICD build to the Percy build.

E.g.

```
import * as percy from '@percy/playwright';

await percy.default(page, 'name', {});
console.log(`Percy build URL: ${await percy.buildUrl()}`);
//https://percy.io/<account id>/<project name>/builds/<build id>
```

**Comment by Robdel12** [comment]
Hey @MartinP-Autopilot! Percy builds aren't created eagerly. Once the build has finished and has been finalized, the build URL is printed at the bottom of the logs. Example logs from a build:

```
$ percy exec -- node script.js
[percy] Percy has started!
[percy] Running "node script.js"
[percy] Snapshot taken "Example Site"
[percy] Stopping percy...
[percy] Finalized build #1: https://percy.io/[your-project]
[percy] Done!
```

**Reply by unlikelyzero** [reply]
It would be great to get that returned from the CLI similar to the way that browserstack returns session details:

https://www.browserstack.com/docs/automate/selenium/get-session-id#get-session-details-of-a-test

I'm not sure about @MartinP-Autopilot 's usecase, but I'd like to return the url of the percy test back onto the PR as a PR Comment.

```
percy exec -- node script.js
percy cli getBuildInfo > percyResult.json

archiveCIartifact percyResult.json
```

Then in my Github Comment
```
console.log('Your Percy Result' + JSON.ToString(percyResult.status) + Json.ToString(percyResult.url))

**Reply by Robdel12** [reply]
For that you can ping the healthcheck endpoint (`http://localhost:5338/percy/healthcheck`), which returns details about the current build. https://github.com/percy/cli/blob/master/packages/core/src/api.js#L38 That's set here, FWIW: https://github.com/percy/cli/blob/master/packages/core/src/percy.js#L170-L172

**Comment by unlikelyzero** [comment]
@MartinP-Autopilot it looks like this is available as of https://github.com/percy/cli/pull/1041
